modules/tinymce/changelog.txt

```diff
@@ -32,6 +32,7 @@ Version 5.7.0 (TBD)
     Fixed incorrect behaviour when editor is destroyed while loading stylesheets #INT-2282
     Fixed figure elements incorrectly splitting from a valid parent element when editing the image within #TINY-6592
     Fixed inserting multiple rows or columns in a table cloning from the incorrect source row or column #TINY-6906
+    Fixed popups incorrectly remaining open when `editor.setProgressState(true)` is called #TINY-6686
```

Comment:
This will also need a changelog entry for the new event, as well as docs. Also, according to the JIRA this should be a changed, not a fixed, as we're changing the behaviour of it.